x-pack/plugins/apm/public/plugin.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import {
+  AppMountParameters,
+  CoreSetup,
+  CoreStart,
+  Plugin,
+  PluginInitializerContext
+} from '../../../../src/core/public';
+import { DEFAULT_APP_CATEGORIES } from '../../../../src/core/utils';
+
+import {
+  PluginSetupContract as AlertingPluginPublicSetup,
+  PluginStartContract as AlertingPluginPublicStart
+} from '../../alerting/public';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { PluginSetupContract as FeaturesPluginSetup } from '../../features/server';
```

Comment:
are we importing the right file here? we don't need the one from `features/public`?

Reply:
you're right, i was importing the wrong type. Also we don't even use the features plugin in the client app, so we could just omit it